srcpkgs/libqb/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'libqb'
+pkgname=libqb
+version=1.0.3
+revision=1
+hostmakedepends="glib-devel pkg-config"
+short_desc="libqb is a library with the primary purpose of providing high-performance, reusable features for client-server architecture, such as logging, tracing, inter-process communication (IPC), and polling."
+only_for_archs="x86_64"
```

Comment:
only_for_archs= is deprecated and archs= should be used instead. I am also not convinced it is actually necessary.